### DISCUSSION TITLE
Redesign Builder's Page (with new UI design scheme)

### DISCUSSION THREAD

**Discussion by ishtails** [BuidlGuidl/batch9.buidlguidl.com#33] [Ideas]
I'd like to redesign the UI of the builder page created by @danitome24 in PR #29 , using the same design pattern that I used for the home page in PR #30. I'm not able to open a new issue for it.. so can anyone assign it to me? :)

**Comment by derrekcoleman** [comment]
Now that #29 is merged, I suggest you open a Draft PR for your redesign
